src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -171,10 +171,7 @@ sylius_inventory:
     resources:
         inventory_unit:
             classes:
-                model: Sylius\Component\Core\Model\InventoryUnit
-        stockable:
-            classes:
-                model: %sylius.model.product_variant.class%
+                model: Sylius\Component\Core\Model\OrderItemUnit
```

Comment:
We should use a parameter here, ``%sylius.model.order_item_unit.class%. Same for ShipmentItem.